Examples/Scripts/Python/itk_seeding.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+import os
+import argparse
+
+import acts
+import acts.examples
+
+from acts.examples import CsvSpacePointReader
+
+u = acts.UnitConstants
+
+
+def runITkSeeding(field, outputDir, s=None):
+
+    # Read input space points from input csv files
+    evReader = CsvSpacePointReader(
+        level=acts.logging.INFO,
+        inputStem="spacepoints",
+        inputCollection="pixel",
+        inputDir="acts/Examples/Scripts/Python/data_itk_seeding/CsvSpacePointsOutput_singleMu_100GeV_300evnts/",
+        outputSpacePoints="PixelSpacePoints",
+    )
+
+    gridConfig = acts.SpacePointGridConfig(
+        bFieldInZ=1.997244311 * u.T,
+        minPt=900 * u.MeV,
+        rMax=320 * u.mm,  # pixel: 320 mm, strip: 1000 mm
+        zMax=3000 * u.mm,
+        zMin=-3000 * u.mm,
+        deltaRMax=280 * u.mm,  # pixel: 280 mm, strip: 600 mm
+        cotThetaMax=27.2899,  # pixel: 27.2899 (4 eta), strip: 900
+        impactMax=2 * u.mm,  # pixel: 2 mm, strip: 20 mm
+        zBinEdges=[
+            -3000.0,
+            -2500.0,
+            -1400.0,
+            -925.0,
+            -450.0,
+            -250.0,
+            250.0,
+            450.0,
+            925.0,
+            1400.0,
+            2500.0,
+            3000.0,
+        ],  # zBinEdges enables non-equidistant binning in z, in case the binning is not defined the edges are evaluated automatically using equidistant binning
+        numPhiNeighbors=1,  # number of phiBin neighbors (plus the current bin) that covers the full deflection of a minimum pT particle
+    )
+
+    seedFinderConfig = acts.SeedfinderConfig(
+        rMax=gridConfig.rMax,
+        deltaRMin=20 * u.mm,
+        deltaRMax=gridConfig.deltaRMax,
+        deltaRMinTopSP=6 * u.mm,  # pixel: 6 mm, strip: 20 mm
+        deltaRMinBottomSP=6 * u.mm,  # pixel: 6 mm, strip: 20 mm
+        deltaRMaxTopSP=280 * u.mm,  # pixel: 280 mm, strip: 3000 mm
+        deltaRMaxBottomSP=120 * u.mm,  # pixel: 120 mm, strip: 3000 mm
+        collisionRegionMin=-200 * u.mm,
+        collisionRegionMax=200 * u.mm,
+        zMin=gridConfig.zMin,
+        zMax=gridConfig.zMax,
+        maxSeedsPerSpM=4,
+        cotThetaMax=gridConfig.cotThetaMax,
+        sigmaScattering=2,
+        radLengthPerSeed=0.09804522341059585,
```

Comment:
quite... surprising precision. the amount of material depends on the forward region and other factors, so in central region is probably less, in more forward regions higher.

what is the reason for having this instead of 0.1?

Reply:
I dumped that value from Athena. I will set it to 0.1, which is probably fine